lib/wibox/init.lua

```diff
@@ -36,6 +37,272 @@ function wibox:set_widget(widget)
     self._drawable:set_widget(widget)
 end
 
+--- Set a declarative widget hierarchy description.
```

Comment:
Since this (and the titlebar version) is duplicate... Does it make sense to have only a single version and link to it? Or can LDoc perhaps include docs from somewhere?

Having multiple versions of this long doc comment is a guarantee for them to get out of sync.